### ISSUE TITLE
Fix Queueing of Ajax calls

### ISSUE THREAD

**Issue by impressivewebs** [impressivewebs/CSS3-Click-Chart#2]
Currently, whenever a feature is chosen, 7 different parts of the page get updated with 7 different Ajax calls. As it is now, and <a href="https://twitter.com/#!/pety7439/status/159012628952395776">as pointed out here</a>, selecting a new feature doesn't cancel any previous uncompleted Ajax calls. So, with a slow connection, they could end up queueing quite a bit, making it clunky.

I couldn't quickly figure out how to fix this (I'm using jQuery's <code>.load()</code> but I would assume I would need to change that), but I will do it at some point. If anyone wants to tackle this, it would be much appreciated.

In short: I need to cancel any in-progress Ajax calls whenever a new feature is clicked.
